charts/parca/README.md

```diff
@@ -59,20 +65,20 @@ helm repo add parca https://parca-dev.github.io/helm-charts
 | agent.extraEnv | list | `[]` | Additional container environment variables for agent |
 | agent.image.pullPolicy | string | `"IfNotPresent"` | Overrides pullpolicy |
 | agent.image.repository | string | `"ghcr.io/parca-dev/parca-agent"` | Overrides the image repository |
-| agent.image.tag | string | `"v0.9.1"` | Overrides the image tag |
+| agent.image.tag | string | `"v0.10.0-rc.1"` | Overrides the image tag |
 | agent.logLevel | string | `"info"` | Agent log level |
 | agent.nodeSelector | object | `{}` | node selector for scheduling agent pods |
 | agent.podAnnotations | object | `{}` | Additional annotations for pods |
```

Comment:
There are also some removed flags such as this. We should clean up as well.

Reply:
I removed the removed `--socket-path` flag.

Reply:
It was not just that one, unfortunately. There are several others. You can find the latest flags in here: https://github.com/parca-dev/parca-agent#configuration